modules/task/backend.go

```diff
@@ -78,31 +74,14 @@ func (b NopBackend) IsRunning() bool {
 type inMemBackend struct {
 	bufQueue  chan []byte
 	isRunning bool
+	isStopped bool
```

Comment:
Yes, that was my comment down below, I replied via email so it didn't show up next to this :(